pagopa/io-app
# feat(Firma con IO): [SFEQS-1535] Update main FCI saga with metadata request

### ⚠️ This PR depends on https://github.com/pagopa/io-app/pull/4473

## Short description
This PR updates FCI saga with metadata request.

## List of changes proposed in this pull request
- Update main saga

## How to test
Using `io-dev-api-server` and a tool like `reacttotron` try to start a signing flow. The state for FCI feature should have `some` metadata pot item.
